### ISSUE TITLE
is the app dead?

### ISSUE THREAD

**Issue by Rodark** [TwitchRecover/TwitchRecover#445]
It wont let me download sub videos

**Comment by ghost** [comment]
The code is outdated... and is prone to bugs so very possible that some functionality will not work properly.

**Comment by Rodark** [comment]
> The code is outdated... and is prone to bugs so very possible that some functionality will not work properly.

oh i see. it just stopped working for me today thats why im asking. when you try to get a sub video it just crashes

**Comment by std-thomas** [comment]
It was working fine for me until today. It just crashes now :(

**Comment by Laritello** [comment]
Run into the same issue. Program crashes because it can't parse the response from the server. Based on stacktrace error happens during feed retrieval.
![image](https://user-images.githubusercontent.com/29901577/222788666-86b2641c-2824-4b28-a160-3fd36792981b.png)

The reason for that is 404 from the server. I checked both types of VODs (public/sub-only) and both requests returned 404. 
![image](https://user-images.githubusercontent.com/29901577/222788323-31e2e4a6-b49e-4297-8dae-f76fc8e42718.png)
![image](https://user-images.githubusercontent.com/29901577/222788581-3bf10ad3-2424-4f0e-8108-0f24a3de3872.png)

Maybe Twitch changed something on the backend?

**Comment by ghost** [comment]
The code is outdated...

**Comment by Laritello** [comment]
I do realise that. But even though code wasn't updated for 2 years it was still working. Also @std-thomas claimed that it was working fine until today. Lately, I'm rarely downloading VODs, so can't confirm nor deny this, but 3 month ago it was still working for me.

And if you're saying that Twitch hasn't changed anything I don't really see how working code suddenly stopped working.
Maybe I'm wrong. In that case my apologies, happy to hear the real reason.

**Comment by ghost** [comment]
This video is unavailable. (Error #5000) - is your reason. 

Twitch is having issues with sub-only vods... nothing has changed though on the code side as your implying.

**Comment by Rodark** [comment]
> This video is unavailable. (Error #5000) - is your reason.
> 
> Twitch is having issues with sub-only vods... nothing has changed though on the code side as your implying.

the video i was checking is definitely available, im currently watching it....

**Comment by Laritello** [comment]
> This video is unavailable. (Error #5000) - is your reason.
> 
> Twitch is having issues with sub-only vods... nothing has changed though on the code side as your implying.

This I can agree with.

**Comment by younesaassila** [comment]
Yup seems like it's dead. It relied on an old Kraken API endpoint; Twitch seems to have shut it down :/

**Comment by churrodude666** [comment]
> This video is unavailable. (Error #5000) - is your reason.
> 
> Twitch is having issues with sub-only vods... nothing has changed though on the code side as your implying.

This appears to be correct. The desktop version gives this error, the mobile app gives 'this broadcast has finished' as the response. I've tried with TR, which instacloses. I've tried with a desktop extension called 'Twitchnosub' which offers merely a large question mark on the screen... 🤣 and Twitchdownloader simply cannot configure. Twitch have either updated to cease programs offering sub only vods for free, or there's a genuine error.

**Comment by wwwLo555** [comment]
Hello! are there any alternatives now ?

**Comment by ghost** [comment]
@wwwLo555 

https://github.com/Shishkebaboo/VodRecovery

https://github.com/tanersb/TwitchRecover

**Comment by churrodude666** [comment]
> @wwwLo555
> 
> https://github.com/Shishkebaboo/VodRecovery
> 
> https://github.com/tanersb/TwitchRecover

Stupid question, but what is the input command to run them?

**Comment by ghost** [comment]
@churrodude666  There is no input command.. you need to install python and the corresponding modules then run them through cmd or an IDE.

The scripts are just basic python scripts they are not CLI based.

**Comment by churrodude666** [comment]
@Shishkebaboo Thank you for the response. What I mean is when you have cmd in the directory, how do you get from that to the scripts output?

**Comment by ghost** [comment]
@churrodude666 

```
cd C:\Users\YOUR_USER\PycharmProjects\VodRecovery
py RecoveryVod.py
```

1. Navigate to the directory where the script is located. 
2. Use the ```py``` command and the name of the script in order to run it.

**Comment by churrodude666** [comment]
> @churrodude666
> 
> ```
> cd C:\Users\YOUR_USER\PycharmProjects\VodRecovery
> py RecoveryVod.py
> ```
> 
> 1. Navigate to the directory where the script is located.
> 2. Use the `py` command and the name of the script in order to run it.

See, I knew I was being stupid. For some reason I kept trying variations of python.py/app.py/etc.. At least this can act as a guide for dummies now. Thank you again
